tests/src/test/scala/system/basic/WskBasicSwift3Tests.scala

```diff
@@ -33,19 +33,20 @@ import common.TestUtils.RunResult
 import spray.json.JsObject
 
 @RunWith(classOf[JUnitRunner])
-class WskBasicSwiftTests
+class WskBasicSwift3Tests
     extends TestHelpers
     with WskTestHelpers
     with JsHelpers {
 
     implicit val wskprops = WskProps()
     val wsk = new Wsk
     val defaultAction = Some(TestUtils.getTestActionFilename("hello.swift"))
-    val currentSwiftDefaultKind = "swift:3"
+    lazy val currentSwiftDefaultKind = "swift:3"
```

Comment:
instead of this use this `ExecManifest.runtimesManifest.resolveDefaultRuntime("swift:default")`
it will returned the actual value based on the configuration